portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -156,6 +156,67 @@ definition {
 		}
 	}
 
+	@description = "LPS-133716 When in Content Dashboard, a column with the Site / assetLibraty header is shown."
+	@priority = "2"
+	test CheckSiteAssertLibraryColumn {
+		task ("Create a Web Content") {
+			JSONWebcontent.addWebContent(
+				content = "Web Content Content",
+				groupName = "Test Site Name",
+				title = "Web Content Title");
+		}
+
+		task ("Check in Content Dashboard if the column is being displayed") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+
+			AssertElementPresent(
+				locator1 = "ContentDashboard#SITE_ASSET_LIBRARY_COLUMN",
+				siteAssetValue = "xxx or Asset Library");
+		}
+	}
```

Comment:
If this is a test with Low priority, it was not needed to be in poshi. However, if you have created just let it be.

Reply:
I created because I would have to practically change only one word from this to the other.